functions.cpp

```diff
@@ -96,4 +96,22 @@ std::string absolute_path(){
     }
 }
 
+std::string strip_accents(std::string str) {
```

Comment:
Where there any tests around this function that need to be moved to utils as well?

Reply:
Thanks, done [here](https://github.com/CanalTP/utils/pull/85/commits/82d891c05c447ea8f017858b13f379f9a1b7b2c3) !